content/06.methods.md

```diff
@@ -298,16 +298,12 @@ The fusion filtering workflow can be found in the [OpenPBTA Analysis repository]
 
 #### Mutational Signatures
 
-We obtained weights for signature sets by applying deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] to consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
-We estimated how many mutations contributed to each signature for each sample using each sample's signature weights.
+We obtained weights (i.e. exposures) for signature sets using the `deconstructSigs` (version `1.8.0`) R package function `whichSignatures()` [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] from consensus SNVs with the BSgenome.Hsapiens.UCSC.hg38 annotations [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
```

Comment:
Did we want to remove the package version here in favor of including it only in the key resources table?

Reply:
We didn't really decide this yet, but can be removed!